docs/setup/security/index.md

```diff
@@ -1,9 +1,79 @@
 # Configuring security in Snow Owl
 
-Snow Owl security features enables you to easily secure your terminology server. You can password-protect your data as well as implement more advanced security measures such as role-based access control and auditing.
+Snow Owl security features enable you to easily secure your terminology server. You can password-protect your data as well as implement more advanced security measures such as role-based access control and auditing.
 
-You can choose the following security realms/identity providers to authenticate your users:
+## Realms
+
+By default Snow Owl comes without any security features enabled and all read and write operations are unprotected.
+To configure a security realm, you can choose from the following built-in identity providers:
 
 * [Configure a file realm](./file.md)
 * [Configure an LDAP realm](./ldap.md)
 
+## Authentication
+
+After configuring at least one security realm, Snow Owl will authenticate all incoming requests to ensure that the sender of the request is allowed to access the terminology server and its contents.
+To authenticate a request, the client must send an HTTP `Basic` or `Bearer` Authorization header with the request.
+The value should be a user/pass pair in case of using `Basic` authentication or a [JWT](https://jwt.io/) token generated by Snow Owl if using the `Bearer` method.
+
+{% hint style="info" %}
+NOTE: It is recommended in production environments that all communication between a client and Snow Owl is performed through a secure connection. 
+{% endhint %}
+
+Snow Owl sends an `HTTP 401 Unauthorized` response if a request needs to be authenticated.
+
+## Authorization
+
+If supported by the security realm, Snow Owl will also check whether an authenticated user is permitted to perform the requested action on a given resource.
+
+Within an organization, roles are created for various job functions. The permissions to perform certain operations are assigned to specific roles. Members or staff (or other system users) are assigned particular roles, and through those role assignments acquire the permissions needed to perform particular system functions. Since users are not assigned permissions directly, but only acquire them through their role (or roles), management of individual user rights becomes a matter of simply assigning appropriate roles to the user's account; this simplifies common operations, such as adding a user, or changing a user's department.
```

Comment:
```suggestion
Within an organization, roles are created for various job functions. The permissions to perform certain operations are assigned to specific roles. Members, staff or other system users are assigned particular roles, and through those role assignments acquire the permissions needed to perform particular system functions. Since users are not assigned permissions directly, but only acquire them through their role (or roles), management of individual user rights becomes a matter of simply assigning appropriate roles to the user's account; this simplifies common operations, such as adding a user, or changing a user's department.
```

Reply:
This part changed only: Members or staff (or other system users)